src/Twitter.php

```diff
@@ -1342,8 +1373,28 @@ public function statusesUserTimeline(array $options = []) : Response
     public function usersLookup($id, array $params = []) : Response
     {
         $path = 'users/lookup';
-        // $params = $this->createUserParameter($id, $params);
-        $params['user_id'] = $id;
+        $params = $this->createUserParameter($id, $params);
```

Comment:
This line was re-instated, as we really need to allow passing either the user identifier or screen name, which was the previous use case; I think we removed it to allow passing a comma-separated list of user identifiers, which is resolved with the addition of `usersLookupMany()`.